client/src/js/components/Bus.js

```diff
@@ -100,6 +106,38 @@ const ModalSection = styled("section")`
   margin: 20px 0px;
 `;
 
+const DatetimeSelection = styled("div")`
+  background: #fff;
+  border-radius: 5px;
+  cursor: pointer;
+`;
+
+const DatePickerWrapper = styled("div")`
+  padding: 0.5em 1em;
+  border: none;
+  border-radius: 5px;
+  display: inline-block;
+  width: 37.5%;
+
+  &:hover {
+    outline: none;
+    background: #ddd;
+  }
+`;
+
+const DatetimeClearButton = styled("div")`
+  padding: 0.5em 1em;
+  border: none;
+  border-radius: 5px;
+  display: inline-block;
+  width: 25%;
+
+  &:hover {
+    outline: none;
+    background: #ddd;
+  }
+`;
```

Comment:
1.`DatePickerWrapper` is technically a button, so please call it `DatePickerButton`
2..`DatetimeClearButton` is almost identical to `DatePickerWrapper`, let's [extend the styles](https://styled-components.com/docs/basics#extending-styles) instead of copying everything. Also, these two components are *buttons*, so the correct HTML tag is "button" instead of "div" 😃

Reply:
I've done the changes!